### ISSUE TITLE
Plans?

### ISSUE THREAD

**Issue by RobLoach** [kodi-game/OpenEmuControllerArt#1]
Do you plan on adding other controller art here? Would be great to complete the set! :+1: 

- 3DS
- Arcade (MAME)
- Dreamcast
- Saturn

Ahh, I see the assets here:
https://github.com/OpenEmu/OpenEmu/tree/master/OpenEmu

Are these used for the addon screenshots/logos?

**Comment by garbear** [comment]
Yes. they are used in the Controller Topology Project https://github.com/kodi-game/controller-topology-project as well as a bunch of artwork I photoshopped from the web
